cfpb/cfgov-refresh
# Hero character count

Hero character count

## Additions

- Added `cfgov/v1/migrations/0060_hero_character_count.py` to add classes to fields.


## Changes

- Modified `cfgov/templates/wagtailadmin/js/hallo-hero-text-count.js` 
- Modified `cfgov/v1/wagtail_hooks.py` to add the hook for the JS file.

## Testing

- Add a Hero to a Sublanding page
- Enter in some text into the heading. Verify that the heading help text is:
  - green when less than 41 characters
  - orange when between 41 and 82 characters
  - red when above 82 characters

- Verify that the body help text is:
   - green when the heading text is less than 41 characters and the body text is less than 165 characters
   - orange when heading text text is less than 41  and the body text is between 165 and 186 characters.
   - red when the heading text is  less than 41 characters and the body text is greater than 186 characters
  - green when the heading text is greater than 41 characters and the body text is less than 108 characters
  - orange when heading text text is greater than 41  and the body text is between 108 and 124 characters.
  - red when the heading text is  greater than 41 characters and the body text is greater than 124 characters.
## Screenshots
- 
<img width="683" alt="screen shot 2017-04-07 at 11 53 19 am" src="https://cloud.githubusercontent.com/assets/1696212/24808175/d4696da4-1b88-11e7-997c-a778e4ef1b43.png">


## Notes

-

## Todos

-

## Checklist
